go/base_client.go

```diff
@@ -942,6 +942,31 @@ func (client *baseClient) MSetNX(ctx context.Context, keyValueMap map[string]str
 	return handleBoolResponse(result)
 }
 
+// Move key from the currently selected database to the database specified by `dbIndex`.
+//
+// See [valkey.io] for details.
+//
+// Parameters:
+//
+//	ctx - The context for controlling the command execution.
+//	key - The key to move.
+//	dbIndex - The index of the database to move key to.
+//
+// Return value:
+//
+//	`true` if `key` was moved, or `false` if the `key` already exists in the destination
+//	database or does not exist in the source database.
+//
+// [valkey.io]: https://valkey.io/commands/move/
+func (client *baseClient) Move(ctx context.Context, key string, dbIndex int64) (bool, error) {
```

Comment:
What is the expected behavior when executing the move command against a cluster in Valkey 7 and 8?

Reply:
As discussed offline. Create the issue https://github.com/valkey-io/valkey-glide/issues/4768 to track this